portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,39 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+    const greetings =
+        ['As you can see from the image, I love golfing⛳️', 
+          'I am from Beijing, China', 
+          '你好，我叫何喆！(My name is Zhe He in Chinese)',
+          'I also loves snow-boarding and skiing (I prefer snowboarding!)',
+          'I love California! The weather is so awesome!'];
+  
+    // Pick a random greeting.
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  
+    // Add it to the page.
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greeting;
+  }
+  
+  function openTag(evt, TabName) {
+    // Declare all variables
+    var i, tabcontent, tablinks;
+  
+    // Get all elements with class="tabcontent" and hide them
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+      tabcontent[i].style.display = "none";
+    }
+  
+    // Get all elements with class="tablinks" and remove the class "active"
```

Comment:
The current comment is just a way of repeating what the code is. You could remove the comment or say something "Update all elements in tablinks so that they are not visible" or something like that. That makes the purpose of the loop clear.